osu.Game.Rulesets.Mania/Mods/ManiaModInvert.cs

```diff
@@ -29,6 +31,9 @@ public class ManiaModInvert : Mod, IApplicableAfterBeatmapConversion
 
         public override Type[] IncompatibleMods => new[] { typeof(ManiaModHoldOff) };
 
+        [SettingSource("Invert Long Notes", "Invert long notes into nothing.")]
+        public BindableBool FullInvert { get; } = new BindableBool();
```

Comment:
needs a better description because i have no idea what this is supposed to be

also i assume the current logic for this in `master` is the same as if this was set to `true`. so either this will have to be changed to default to true, or all existing scores with invert in the database will need to be updated for correct playback. not sure what about replays either. i guess they'd just be hard-broken?